src/store.js

```diff
@@ -101,8 +101,11 @@ export class PortalStore {
   }
 
   @action.bound
-  fetchProfileCharacters(owner){
-    return this.api.get(`/characters/?owner=${owner}`).then(response => response.data);
+  fetchProfileCharacters(owner,dead){
```

Comment:
aaaand space

Reply:
Also... how about simply moving this to the separate method called `fetchProfileDeadCharacters`? For now it's fine. If future filters show up, we should think this through.